framework/src/types.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as liskP2p from '@liskhq/lisk-p2p';
```

Comment:
why not `import { p2pTypes : { P2PConfig } } from '@liskhq/lisk-p2p';`

Reply:
I could do 
```
import { p2pTypes } from '@liskhq/lisk-p2p';
p2pTypes.P2PConfig
```